netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -183,101 +182,101 @@
-      <Compile Include="src\System.Reflection.Emit\MonoArrayMethod.cs" />
-      <Compile Include="src\System.Reflection.Emit\ParameterBuilder.cs" />
-      <Compile Include="src\System.Reflection.Emit\PropertyBuilder.cs" />
-      <Compile Include="src\System.Reflection.Emit\PropertyOnTypeBuilderInst.cs" />
-      <Compile Include="src\System.Reflection.Emit\SignatureHelper.cs" />
-      <Compile Include="src\System.Reflection.Emit\TypeBuilder.cs" />
-      <Compile Include="src\System.Reflection.Emit\TypeBuilderInstantiation.cs" />
-      <Compile Include="src\System.Reflection.Emit\UnmanagedMarshal.cs" />
-      <Compile Include="src\System.Reflection.Metadata\AssemblyExtensions.cs" />
-      <Compile Include="src\System.Resources\ManifestBasedResourceGroveler.cs" />
-      <Compile Include="src\System.Runtime\GCSettings.cs" />
-      <Compile Include="src\System.Runtime.CompilerServices\DependentHandle.cs" />
-      <Compile Include="src\System.Runtime.CompilerServices\JitHelpers.cs" />
-      <Compile Include="src\System.Runtime.CompilerServices\RuntimeHelpers.cs" />
-      <Compile Include="src\System.Runtime.CompilerServices\RuntimeFeature.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\CriticalHandle.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\GCHandle.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\Marshal.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\MarshalAsAttribute.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\NativeLibrary.cs" />
-      <Compile Include="src\System.Runtime.InteropServices\SafeHandle.cs" />
-      <Compile Include="src\System.Runtime.Loader\AssemblyLoadContext.cs" />
-      <Compile Include="src\System.Runtime.Loader\AssemblyDependencyResolver.cs" />
-      <Compile Include="src\System.Runtime.Remoting.Contexts\Context.cs" />
-      <Compile Include="src\System.Security\DynamicSecurityMethodAttribute.cs" />
-      <Compile Include="src\System.Threading\Interlocked.cs" />
-      <Compile Include="src\System.Threading\Monitor.cs" />
-      <Compile Include="src\System.Threading\Overlapped.cs" />
-      <Compile Include="src\System.Threading\PreAllocatedOverlapped.cs" />
-      <Compile Include="src\System.Threading\StackCrawlMark.cs" />
-      <Compile Include="src\System.Threading\Thread.cs" />
-      <Compile Include="src\System.Threading\ThreadPool.cs" />
-      <Compile Include="src\System.Threading\LowLevelLock.cs" />
-      <Compile Include="src\System.Threading\LowLevelSpinWaiter.cs" />
-      <Compile Include="src\System.Threading\WaitHandle.cs" />
+      <Compile Include="src\System\Globalization\GlobalizationMode.Mono.cs" />
+      <Compile Include="src\System\Collections\Generic\ArraySortHelper.cs" />
+      <Compile Include="src\System\Collections\Generic\Comparer.cs" />
+      <Compile Include="src\System\Collections\Generic\EqualityComparer.cs" />
```

Comment:
This looks better but still need more tweaks for example this one is partial hence should be .Mono.cs